CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- [BREAKING] - Change onTapImage to OnTapImage to remove analyser warning
+- Removing lock file from repository - adding it to gitignore
+- Updating ios version for example project to 11 to support better player
+- Regenerating example project with flutter create command
+- Removing unused video_player dependency
```

Comment:
I think we can divide the Changelog a bit more, to use `### Removed` for `Removing unused video_player dependency` and the lock file.

Reply:
done